packages/dashboard/src/components/global-shortcuts/global-shortcuts.tsx

```diff
@@ -0,0 +1,295 @@
+
+    // Trigger click on the create/add/upload button in the current page
+    // Strategy: Find buttons by common patterns in text content (works across languages)
+    const buttons = Array.from(document.querySelectorAll('button'));
+
+    const actionButton = buttons.find((btn) => {
+      const text = btn.textContent?.toLowerCase() || '';
+      const ariaLabel = btn.getAttribute('aria-label')?.toLowerCase() || '';
+
+      // Check for common action words from actual translations
+      const actionWords = [
+        // English
+        'add',
+        'create',
+        'new',
+        'upload',
+        'register',
+        // Spanish
+        'añadir',
+        'agregar',
+        'crear',
+        'nuevo',
+        'subir',
+        'registrar',
+        // Swedish
+        'lägg till',
+        'skapa',
+        'ny',
+        'ladda upp',
+        'registrera',
+        // German
+        'hinzufügen',
+        'erstellen',
+        'neu',
+        'hochladen',
+        'registrieren',
+        // French
+        'ajouter',
+        'créer',
+        'nouveau',
+        'télécharger',
+        'enregistrer',
+        // Chinese
+        '添加',
+        '创建',
+        '新',
+        '上传',
+        '注册',
+        // Arabic
+        'إضافة',
+        'إنشاء',
+        'جديد',
+        'رفع',
+        'تسجيل',
+        // Korean
+        '추가',
+        '만들기',
+        '새',
+        '업로드',
+        '등록',
+        // Japanese
+        '追加',
+        '作成',
+        '新しい',
+        'アップロード',
+        '登録',
+      ];
+
+      return actionWords.some(
+        (word) => text.includes(word) || ariaLabel.includes(word)
+      );
+    });
+
+    if (actionButton) {
+      (actionButton as HTMLButtonElement).click();
+    } else if (props.onCreateResource) {
+      props.onCreateResource();
+    }
+  };
+
+  onMount(() => {
+    // Show shortcuts legend - using ? which is shift+/ on most keyboards
+    registerShortcut('show-shortcuts', {
+      key: '?',
+      shift: true,
+      description: () => t('shortcuts.showShortcutsLegend'),
+      category: 'global',
+      action: () => props.onShowShortcuts(),
+    });
+
+    // Global Search - moved to global category
+    registerShortcut('global-search', {
+      key: 'F',
+      ctrl: true,
+      description: () => t('shortcuts.search'),
+      category: 'global',
+      action: () => {
+        const searchInput = document.querySelector(
+          'input[type="search"], input[placeholder*="Search"], input[placeholder*="search"]'
+        ) as HTMLInputElement;
+        if (searchInput) {
+          searchInput.focus();
+        }
+      },
+    });
+
+    // ESC to close dialogs and exit input fields
+    registerShortcut('close-dialog', {
+      key: 'Escape',
+      description: () => t('shortcuts.closeDialog'),
+      category: 'global',
+      action: () => {
+        // If focus is in an input field, blur it
+        const activeElement = document.activeElement as HTMLElement;
```

Comment:
We do not need this as dialogs already are closed using ESC as this is part of their implementation, so in this case the action should be a noop.